docs/content/install.md

```diff
@@ -129,3 +131,64 @@ All of the Porter-authored plugins are published to `https://cdn.porter.sh/plugi
 
 [releases]: https://github.com/getporter/porter/releases
 
+
+
+# Install Script Parameters
+
+The installation scripts provide the following parameters. Paramters can be specified with environment variables for the macOS and Linux scripts, and on Windows they are named parameters in the script.
```

Comment:
```suggestion
The installation scripts provide the following parameters. Parameters can be specified with environment variables for the macOS and Linux scripts, and on Windows they are named parameters in the script.
```

Reply:
I fixed a couple places with that same typo. I really have trouble with typing that word! 😊